.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: Code Quality
+
+on:
```

Comment:
I personally prefer the `on: [push, pull_request]` syntax but its ok.

Reply:
we can change it, it looks more neat in that way tho